packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Providers/NamedTypeSymbolProviderTests.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.Generator.CSharp.Primitives;
+using Microsoft.Generator.CSharp.Providers;
+using NUnit.Framework;
+
+namespace Microsoft.Generator.CSharp.Tests.Providers
+{
+    public class NamedTypeSymbolProviderTests
+    {
+        private NamedTypeSymbolProvider _namedTypeSymbolProvider;
+        private NamedSymbol _namedSymbol;
+
+        public NamedTypeSymbolProviderTests()
```

Comment:
do we need the [SetUp] attribute?

Reply:
No I am just doing this in the ctor since none of the tests need a unique / fresh typefactory or outputlibrary to work with.

If you need a fresh TF for each test this would all move to a setup method.